glTF/glTF-Property-Texture-Sandcastle.js

```diff
@@ -0,0 +1,60 @@
+const viewer = new Cesium.Viewer("cesiumContainer");
+
+// Create the tileset, and move it to a certain position on the globe
+let tileset = viewer.scene.primitives.add(
+  new Cesium.Cesium3DTileset({
+    url: `http://localhost:8003/glTF/EXT_structural_metadata/SimplePropertyTexture/tileset.json`,
+    debugShowBoundingVolume: true,
+  })
+);
+tileset.modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(
+  Cesium.Cartesian3.fromDegrees(-75.152325, 39.94704, 0)
+);
+const offset = new Cesium.HeadingPitchRange(
+  0,
+  Cesium.Math.toRadians(-22.5),
+  4.0
+);
+viewer.zoomTo(tileset, offset);
+
+// Create a custom (fragment) shader that accesses the metadata value with the
+// given property name, normalizes it to a value in [0,1] based on the given
+// source range, and uses that value as the brightness for the fragment.
+function createShader(propertyName, sourceMin, sourceMax) {
+  const shader = new Cesium.CustomShader({
+    fragmentShaderText: `
+      void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material)
+      {
+        float value = float(fsInput.metadata.${propertyName});
+        float range = float(${sourceMax}) - float(${sourceMin});
+        float brightness = (value - float(${sourceMin})) / range;
```

Comment:
Just wondering, have you considered using the `normalized` property, and/or the `offset/scale` options? these transformations are applied before properties are added to `fsInput.metadata`.

Reply:
I considered this. I _think_ that it would add some complexity, but maybe I overlooked some easy and elegant solution...

Reply:
Looking at the `SimplePropertyTexture` example again, I see that you do use a mix of normalized and unnormalized properties. So I think it makes sense to leave this as-is.